app/main.py

```diff
@@ -1,3 +1,56 @@
+from typing import Union
+
+
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Union) -> object:
+        if isinstance(other, (int, float)):
+            return Distance(self.km + other)
+        else:
```

Comment:
remove unnecessary else

Reply:
if I do this I get AssertionError: Result of sum of Distance instances should be Distance instance

Reply:
def __add__(self, other: Union) -> object:
        if isinstance(other, (int, float)):
            return Distance(self.km + other)
        return Distance(self.km + other.km)